.gitignore

```diff
@@ -14,4 +14,4 @@ PennCourses/settings/static/
 *.sqlite3
 *-test.json
 celerybeat-schedule
-pcr-backup
+pcr-backup*
```

Comment:
Why is the star necessary?

Reply:
Mainly so we can have multiple backup directories. On my machine, `pcr-backup` contains all PCR data up until 2018A -- I made a new directory called `pcr-backup-semester` to test this feature out, wanted to make sure it was also in the gitignore

Reply:
Makes sense